packages/cli/src/commands/rewards/show.ts

```diff
@@ -24,6 +24,7 @@ export default class Show extends BaseCommand {
 
   static flags = {
     ...BaseCommand.flagsWithoutLocalAddresses(),
+    estimate: flags.boolean({ description: 'Estimate voter rewards from current state' }),
```

Comment:
Because non-archive is the default configuration for a node, the defaults for this command should not rely on it. Instead of adding this estimate flag to opt-out of archival state use, the option should be opt-in. My suggestion would be to overload the `epochs` flag by making it default to `0` or `undefined`, in which case only current state will be used, and if the user specifies a number there, then use archival state. It should also be explicit in the command or flag description that the target node must have the desired state in it's archives.

Reply:
Added some explanatory error handling.  The defaults work for 85 minutes after each epoch.  After that an archive node is needed to determine exact voter rewards (archival state issue only applies when --voter supplied).  If you haven't changed your votes, then your current share of votes for a group should be a good estimation of your share 90 minutes or 23 hours ago.  But we can't always show exact information for the latest epoch. I think an `estimate` flag makes that clear.

Reply:
I like the error handling! 👍